docker-compose.yml

```
@@ -0,0 +1,8 @@
+services:
+  bot:
+    build: .
+    volumes:
+     - .:/root/discord-php
+volumes:
+  bot:
+    driver: local
```

```suggestion
    driver: local

```
End files with empty lines :^)